ctms/config.py

```diff
@@ -19,6 +19,9 @@ class LogLevel(str, Enum):
 
 class Settings(BaseSettings):
     db_url: PostgresDsn
+    db_pool_size: int = 10
+    db_max_overflow: int = 20
+    db_pool_timeout_in_seconds: int = 45
```

Comment:
These do not match the defaults of SQLA, how did you pick these values?

Reply:
After viewing:
https://my.papertrailapp.com/groups/24734221/events?q=exception&focus=1448826046895013939&selected=1448826046895013939
```
TimeoutError('QueuePool limit of size 5 overflow 10 reached, connection timed out, timeout 30')
```
and looking into: https://docs.sqlalchemy.org/en/13/errors.html#queuepool-limit-of-size-x-overflow-y-reached-connection-timed-out-timeout-z

The defaults were pool_size=5, max_overflow=10, pool_timeout=30; since these values were not working and were resulting in both QueuePool limit's and timeout's being hit--upping these values made sense to me. The pool_size and max_overflow were doubled and the timeout was increased by 50%. 

Do you suggest a particular approach for picking these values?

Reply:
I'm ok, hence the approval ;) 

I didn't have the backstory, and just wanted to underline that there were 2 intentions here: making it configurable, and increasing defaults

Reply:
That makes sense, those are the two things occurring; thanks for calling it out!